src/appshell/view/dockwindow/internal/dockbase.cpp

```diff
@@ -557,11 +557,9 @@ void DockBase::applySizeConstraints()
     m_dockWidget->setMaximumSize(maximumSize);
 
     if (KDDockWidgets::FloatingWindow* window = m_dockWidget->floatingWindow()) {
-        window->setMinimumSize(minimumSize);
```

Comment:
If these lines aren't removed, they will limit the size of the opened floating window to be smaller than it was last sized for.